languages/python/reference/exercise-concepts/markdown.md

```diff
@@ -0,0 +1,48 @@
+# Concepts for markdown
+
+[Example Implementation](https://github.com/exercism/python/blob/master/exercises/markdown/example.py)
+
+## General  
+
+
+- functions: This exercise requires a re-write of pre-esiting code that uses functions to parse passed-in text in _markdown_.  Tests expect a function named `parse` that can be called to transform the _markdown_ formatted text and return HTML formatted text.
+
+- function arguments:  The example solution uses multiple functions that take various function arguments to operate on different pieces of the passed in markdown string.
```

Comment:
The example solution is pretty hideous in that regard ... this is another exercise where  we might want to deviate from the example.py files, which were never written to be particularly good or sensible solutions.

Reply:
Added both a `re` only and a `re` plus Python example.  Left the example.py, but wow...it really needs to die.  Vote for one of the other two as the new `example.py`.

> - function arguments:  The example solutions use functions that take function arguments to operate on passed in markdown strings.